themes/default/content/blog/five-years-of-infrastructure-as-code-part-one/index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "Five Years of Infrastructure as Code: Why?"
```

Comment:
```suggestion
title: "Five Years of Infrastructure as Code"
```

I had imagined maybe doing two blog posts:
1) The Five Years of Infrastructure as Code post: it's just an index basically.
2) Each of the subsequent posts as an independent thing that the index is updated to link to.

This would let us have good titles for each of the (2)s without needing to prefix like this. If we keep it combined right now, I'd probably just keep the first post simple and say "Five Years of Infrastructure as Code."